README.md

```diff
@@ -132,6 +132,7 @@ subjective "from more to less liquid control-ly" order.
 | AIO liquid cooler  | [NZXT Kraken X42, X52, X62, X72](docs/kraken-x2-m2-guide.md) | <sup>_h_</sup> | 1.11.1 |
 | AIO liquid cooler  | [NZXT Kraken X53, X63, X73](docs/kraken-x3-z3-guide.md) | <sup>_h_</sup> | 1.11.1 |
 | AIO liquid cooler  | [NZXT Kraken Z53, Z63, Z73](docs/kraken-x3-z3-guide.md) | <sup>_eh_</sup> | 1.11.1 |
+| AIO liquid cooler  | [NZXT Kraken 2023 (Elite, Standard)](docs/kraken-x3-z3-guide.md) | <sup>_eh_</sup> | - |
```

Comment:
The hwmon driver does not support these product IDs right now.

Reply:
A related (but less important) note: "NZXT Kraken 2023 Standard, Elite" would better fit in with the rest of the table.

The MRLV for a new device is "git".